neurokit2/rsp/rsp_process.py

```diff
@@ -78,19 +78,15 @@ def rsp_process(rsp_signal, sampling_rate=1000, method="khodadad2018"):
     rsp_cleaned = rsp_clean(rsp_signal, sampling_rate=sampling_rate, method=method)
 
     # Extract, fix and format peaks
-    peak_signal, info = rsp_peaks(
-        rsp_cleaned, sampling_rate=sampling_rate, method=method, amplitude_min=0.3
-    )
+    peak_signal, info = rsp_peaks(rsp_cleaned, sampling_rate=sampling_rate, method=method, amplitude_min=0.3)
     info["sampling_rate"] = sampling_rate  # Add sampling rate in dict info
 
     # Get additional parameters
     phase = rsp_phase(peak_signal, desired_length=len(rsp_signal))
     amplitude = rsp_amplitude(rsp_cleaned, peak_signal)
-    rate = signal_rate(
-        info["RSP_Troughs"], sampling_rate=sampling_rate, desired_length=len(rsp_signal)
-    )
+    rate = signal_rate(info["RSP_Troughs"], sampling_rate=sampling_rate, desired_length=len(rsp_signal))
     symmetry = rsp_symmetry(rsp_cleaned, peak_signal)
-    rvt = rsp_rvt(rsp_cleaned, sampling_rate=sampling_rate, silent=True)
+    rvt = rsp_rvt(rsp_cleaned, method="harrison2021", sampling_rate=sampling_rate, silent=True)
```

Comment:
I think ideally this could be set with a keyword argument to `rsp_process()` such as `method_rvt`, similar to `method_cleaning` and `method_peaks` in `ppg_methods()`

Reply:
Good catch

Reply:
@DominiqueMakowski would be interested in your input here, from what I understood from your comment before (https://github.com/neuropsychology/NeuroKit/pull/750#issuecomment-1316086324) you wanted there to be `method_rvt()` so that the user can specify `"harrison2021"`, `"power2020"`, or `"birn2006"`?

Reply:
yes, I'd add `method_rvt` as an arg to rsp_methods so that it can be changed

Reply:
Hopefully the latest addition is how you intended it.

This should encompass all requested changes for the progress in this PR.